windnode_abw/model/region/tools.py

```diff
@@ -481,3 +481,37 @@ def calc_heat_pump_cops(t_high, t_low, quality_grade,
                 cops = cops + [quality_grade * t_h / (t_h - t_l)]
 
     return cops
+
+
+def calc_dsm_cap_up(data, ags, mode='simple'):
+    """calculates the correct format of dsm input"""
+    demand = data['Lastprofil', ags]
+
+    if mode == 'simple':
```

Comment:
I find the possible values of `mode` param a little misleading. It's not obvious what 'simple' or 'advanced' refers to, sounds more like a methodical approach... I'd rather use something like 'flex_min', 'flex_max' or similar..